src/hooks/useTreasury.ts

```diff
@@ -73,7 +73,7 @@ const useTreasury = () => {
 
   const totalYUsdValue = useMemo(() => {
     const yamYUsdValue = yamTwap && yamBalance ? yamTwap * yamBalance : 0;
-    return yUsdBalance ? yUsdBalance + yamYUsdValue : yamYUsdValue;
+    return (yUsdBalance ? yUsdBalance + yamYUsdValue : yamYUsdValue) + 810000;
```

Comment:
this is wrong, we worked on the treasury history object for this specific purpose. please use it